src/Components/v2/CarouselV3.tsx

```diff
@@ -63,7 +68,8 @@ type ArrowProps = (props: {
 
 export class Carousel extends React.Component<CarouselProps> {
   static defaultProps = {
-    height: 300,
+    height: "300px",
+    isOneSlide: false,
```

Comment:
Also, how many slides a carousel has can be inferred from `data.length` vs a declaration

Reply:
It's the visible slides not the length of the carousel

Reply:
Maybe I can make that more explicit in the naming of this variable

Reply:
Ahh i see; yup, just a naming issue 👍 That said, this sort of thing is pretty low-level; sure there's not something within Flickity that we can piggyback on?

Reply:
We need to know before flickity is initialized so to only render one image for SSR, so we wouldn't be able to use flickity's API.

Reply:
I guess what i mean is something like [`slidesToShow`](https://react-slick.neostack.com/docs/api#slidesToShow) as seen in `react-slick`; this kind of detail can quickly lead to complexity within the wrapper so I think it might be worth thinking about a bit before adding. We're working around an SSR issue, rather than defining behaviors for the consumer of the carousel. If we _didn't_ need to SSR render then this prop would be unnecessary. 

Let's give this a think... the snippet sent last night was a bit of a hack 🤔 🤔

Reply:
```const carouselImages = typeof window === "undefined" && oneSlideVisible ? [data[0]] : data```

We need to do something like this otherwise if the carousel has more than one image it will only render the first image on page load.

Reply:
Thoughts on https://github.com/metafizzy/flickity/issues/526?

Reply:
We can't use Flickity's API, because this is pre-flickity loading. This is just for SSR so we don't have that double image in one slide like we were seeing in SSR.